src/viewer.js

```diff
@@ -33,6 +33,7 @@ if (typeof OCA.Viewer === 'undefined') {
 	OCA.Viewer.registerHandler({
 		id: 'text',
 		mimes: [...openMimetypesMarkdown, ...openMimetypesPlainText],
+		// Would be good to be able to lazyload that
 		component: ViewerComponent,
```

Comment:
We cannot really lazy load a viewer component yet, so it would be nice if one could just pass a callback here for a lazy import